### PR TITLE
[DO NOT MERGE] Updating Fitbit connection doc for OAuth2

### PR DESCRIPTION
**Pull request by jeffreylees**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: articles/connections/social/fitbit.md [hunk 1]
```diff
@@ -4,15 +4,17 @@ image: /media/connections/fitbit.png
 seo_alias: fitbit
 ---
 
-# Obtain a *Consumer Key* and *Consumer Secret* for Fitbit
+# Obtain a *Client ID* and *Client Secret* for Fitbit
 
-To configure a Fitbit oAuth connection, you will need to register a new application in Fitbit.
+**Fitbit has ended support for OAuth 1.0. New connections with Fitbit will use OAuth 2.0. Please see the following for reference on OAuth 2.0 at Fitbit: [https://dev.fitbit.com/docs/oauth2/](https://dev.fitbit.com/docs/oauth2/).**
```
Comment (Catografix) [inline articles/connections/social/fitbit.md]: I think a warning panel works best for this

::: panel-warning Fitbit has ended support for OAuth 1.0
New connections with Fitbit will use OAuth 2.0. Please see the following for reference on OAuth 2.0 at Fitbit: [https://dev.fitbit.com/docs/oauth2/](https://dev.fitbit.com/docs/oauth2/)
:::

**Comment by ziluvatar** [comment]
It looks nice, I'll let you know when we can merge, dashboard is almost done.

**Comment by Catografix** [comment]
Preview OK: https://auth0-docs-content-pr-1602.herokuapp.com/docs/connections/social/fitbit

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Pull request merged**